activiti-cloud-starter-runtime-bundle/src/test/resources/processes/activitySignalEventsInSubprocess.bpmn20.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns="http://www.omg.org/spec/BPMN/20100524/MODEL"
+  xmlns:activiti="http://activiti.org/bpmn" targetNamespace="Examples">
+
+  <signal id="mySignal" name="mySignal"/>
+
+  <process id="signalSubProcess" name="signalSubProcess">
+
+    <startEvent id="theStart" />
+    <sequenceFlow id="flow1" sourceRef="theStart" targetRef="shipOrder" />
+
+    <userTask id="shipOrder" name="Wait" />
+
+    <sequenceFlow id="flow2" sourceRef="shipOrder" targetRef="theEnd" />
+
+    <endEvent id="theEnd" />
+
+ 	<sequenceFlow id="flow3" sourceRef="theStart" targetRef="eventSubProcess" />
+ 
+	<subProcess id="eventSubProcess" name="eventSubProcess" triggeredByEvent="true">

```

Comment:
You should remove `triggeredByEvent="true"`, as it's not triggered by an event anymore.